### DISCUSSION TITLE
Best way to define one way references

### DISCUSSION THREAD

**Discussion by cysabi** [prisma/prisma#13430] [Q&A] [answered]
Hello, I'm looking for some insight on how to map database relationships in Prisma (MongoDB) for a certain kind of use case.

Imagine I'm making a linked-in clone, where I have 3 models, `User`, `Company`, and `Experience`.

A user can have multiple Experience listed on their page, so that's obviously a one-to-many relationship.

Each experience also refers to a Company, but **Companies do not need to know anything about Experience objects.**

Is there a way to store this reference that doesn't require the Company model to include a back reference to every Experience object that references the Company?

Is the only way to store this as a one-to-many relationship (where companies can have multiple experience) and having a lot of unused relations.
Is there a better way to map this data as a one-way relationship?

What about if an Experience refers multiple Companies?

**Comment by nurul3101** [comment, marked as answer]
Hey @LeptoFlare 👋 

It's not possible to have one-way references while defining relations between two models. You need to define relation fields on both sides of the model, it's a requirement from Prisma's end. So for the relation between the `Companies` and `Experience` model you would need to define relation fields on both of these models.

**Comment by ludralph** [comment]
Hi there,

To keep our discussions organized and focused on the most relevant topics, we’re reviewing and tidying up our backlog. As part of this process, we’re closing discussions that have already been marked as answered but remain open.

If this discussion still requires further input or clarification, feel free to reopen it or start a new one with updated details. Your contributions are invaluable to the community, and we’re here to help!

For more details about our priorities and vision for the future of Prisma ORM, check out our latest blog post: https://www.prisma.io/blog/prisma-orm-manifesto.

Thank you for your understanding and ongoing support of the Prisma community!
